features/integration/compile.feature

```diff
@@ -20,3 +21,11 @@ Feature: Compile
       | "programs/one.teal"             | "programs/one.teal.tok"             |
       | "programs/zero.teal"            | "programs/zero.teal.tok"            |
       | "programs/abi_method_call.teal" | "programs/abi_method_call.teal.tok" |
+
+  @compile.sourcemap
+  Scenario Outline: Teals return a valid Source Map
```

Comment:
nit: I wonder why the previous tests here chose to write `Teals`

Reply:
Agree - seems a bit strange to me.  At least for the highlighted scenario, I imagined something like "Algod compiling TEAL returns a valid source map" or similar.  

I don't feel strongly.